contracts/USM.sol

```diff
@@ -18,22 +18,24 @@ contract USM is ERC20 {
 
     address public oracle;
     FUM public fum;
-    uint public latestFumPrice;                               // default 0
+    uint public minFumBuyPrice;                               // in units of ETH. default 0
 
     uint public constant WAD = 10 ** 18;
-    uint constant public MIN_ETH_AMOUNT = WAD / 1000;         // 0.001 ETH
-    uint constant public MIN_BURN_AMOUNT = WAD;               // 1 USM
-    uint constant public MAX_DEBT_RATIO = WAD * 8 / 10;       // 80%
+    uint public constant MIN_ETH_AMOUNT = WAD / 1000;         // 0.001 ETH
+    uint public constant MIN_BURN_AMOUNT = WAD;               // 1 USM
+    uint public constant MAX_DEBT_RATIO = WAD * 8 / 10;       // 80%
 
-    event LatestFumPriceChanged(uint previous, uint latest);
+    int public constant BUY = 1;
```

Comment:
Instead you should use an `enum`: https://solidity.readthedocs.io/en/v0.6.7/types.html#enums

`enum Side {Buy, Sell}`

By default, `Side.Buy == 0` and `Side.Sell == 1`, because enum members are numbered sequentially. When you interact with them from outside a contract you pass the numerical value, although the user should be oblivious to that.

Reply:
👍